conftest.py

```diff
@@ -67,10 +67,12 @@ def pytest_sessionfinish(session, exitstatus):
 
 OutputChecker = doctest.OutputChecker
 
+
 class CustomOutputChecker(OutputChecker):
     def check_output(self, want, got, optionflags):
-        if IGNORE_RESULT and optionflags:
+        if IGNORE_RESULT & optionflags:
```

Comment:
@sgugger - I've dug a bit into this. `IGNORE_RESULTS` is actually not a bool but an integer so as it was coded now **all** doc tests were ignored. We need to use the bitwise and here I think (tested it locally and it works again now).

Reply:
Ah, thanks for checking and fixing! I had tested it locally but I must have messed something up.